src/xml-parsers.js

```diff
@@ -236,7 +246,12 @@ export function parseListObjects(xml) {
   }
   let isTruncated = false
   let nextMarker, nextVersionKeyMarker
-  const xmlobj = parseXml(xml)
+  const fxp = new XMLParser({
```

Comment:
maybe move this outside function?

Reply:
Done

Reply:
parser can be re-used, not very necessary to create new one for each parsing